src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1439,4 +1439,32 @@ public static void setFilteredGenotypeToNocall(final VariantContextBuilder build
         // update genotypes
         builder.genotypes(genotypes);
     }
+
+    /**
+     * Calculate the Genotype Quality (GQ) by subtracting the second smallest Phred-scaled likelihoods (PL) first smallest.
+     *
+     * @param plValues  array of PL values
+     * @return          the genotype quality corresponding to the PL values
+     */
+    public static int calculateGQFromPLs(final int[] plValues) {
+        Utils.nonNull(plValues);
+        Utils.validateArg(plValues.length >= 2, () -> "Array of PL values must contain at least two elements.");
+
+        int first = plValues[0];
+        int second = plValues[1];
+        if (first > second) {
+            second = first;
+            first = plValues[1];
+        }
+        for (int i = 2; i < plValues.length; i++) {
+            final int candidate = plValues[i];
+            if (candidate >= second) continue;
```

Comment:
I think we are typically more strict about adherence to the Google Java Style guide in gatk-protected, so I would add braces to all the blocks here (even if optional).